bench/wtperf/runners/checkpoint-stress.wtperf

```diff
@@ -1,23 +1,26 @@
 # A stress configuration to create long running checkpoints while doing a lot
```

Comment:
The change to this file is a tidy up related to WT-2868, it's not particularly relevant for this change.